cleanlab/object_detection/rank.py

```diff
@@ -103,6 +105,9 @@ def get_label_quality_scores(
        swapped examples, bad location examples, and overlooked examples.
        It is important to ensure that the weights are non-negative values and that their sum equals 1.0.
 
+    overlapping_label_check : bool, default = True
```

Comment:
Nice job integrating this param into rest of code. Maybe for the 3 docstring, say what happens if set to false. Something like:
If True, annotated examples with more than one label associated with them are intentionally given a low swap score. Set to ``False`` to assign annotated examples with more than one label associated with them their expected scores instead.

(remember to do this in all 3 docstrings)

Reply:
I added suggestion of this in one of the docstrings, you can copy it for the others

Reply:
Added